src/store/poolSlice.ts

```
@@ -23,6 +23,38 @@ export interface PoolState {
+    if(poolFetchResponse) {
+      const newPool = {
+        poolId,
+        round: 0,
+        tokA: poolFetchResponse.tokA,
+        tokB: poolFetchResponse.tokB,
+      }
+      await db.table("pools").put(newPool);
+      return newPool;
+    }
+  } catch (error) {
+    throw error;
```

Biome

[error] The catch clause that only rethrows the original error is redundant.

These unnecessary catch clauses can be confusing. It is recommended to remove them.

(lint/complexity/noUselessCatch)